app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageViewModel.kt

```diff
@@ -811,8 +811,9 @@ class NewMessageViewModel @Inject constructor(
         }
     }
 
-    private fun Draft.updateDraftAttachmentsFromLiveData() {
-        val updatedAttachments = attachmentsLiveData.valueOrEmpty().map { uiAttachment ->
+    private fun Draft.updateDraftAttachmentsFromLiveData(uiAttachments: List<Attachment>) {
```

Comment:
We could rename the method with something along the lines of `updateDraftAttachmentsWithLiveData`